core/templates/dev/head/pages/topic-viewer-page/topic-viewer-page.directive.html

```diff
@@ -21,6 +26,9 @@
   <div ng-if="$ctrl.activeTab === 'practice'" class="oppia-practice-tab">
     <practice-tab topic-name="$ctrl.topicName"></practice-tab>
   </div>
+  <div ng-if="$ctrl.activeTab === 'skill'" class="oppia-skill-tab">
```

Comment:
Do we have a constant for "skill" if yes then let's use it as I see "skill" string is used in multiple places in this file?

Reply:
This part is removed due to @aks681 's comment.